src/units/abilities.cpp

```diff
@@ -1168,6 +1168,48 @@ void attack_type::modified_attacks(unsigned & min_attacks,
 	}
 }
 
+//Functions used for change damage_type list with damage
+static std::vector<std::string> vector_type(unit_ability_list& abil_list, const std::string& type)
```

Comment:
`abil_list` can be const here. Also, please change the function name to `damage_type_list`.